### ISSUE TITLE
Potential Data Leakage in Dataset Split

### ISSUE THREAD

**Issue by Xxiong01** [vranlee/SU-T#4]
Hi, I have a question about how the datasets are split.
I noticed that we are using:
Training set: train.json
Validation set: val_half.json
The documentation states that  is a combination of  and . This means the validation set is entirely contained within the training set.train.jsontrain_half.jsonval_half.json
Doesn't this approach create a risk of data leakage?

**Comment by vranlee** [comment]
Hi. You should only use the paired xxx-half.json for ablation exps.
The train.json includes the ENTIRE training set, used for comparative experiments, and the test set used for it.
